ghost/core/core/server/update-check.js

```diff
@@ -17,7 +17,7 @@ const UpdateCheckService = require('@tryghost/update-check-service');
  * @returns {Promise<any>}
  */
 module.exports = async ({rethrowErrors = false} = {}) => {
-    const allowedCheckEnvironments = ['development', 'production'];
+    const allowedCheckEnvironments = ['testing', 'testing-mysql', 'development', 'production'];
```

Comment:
I wonder if adding the update check on all `testing-*` environments is desired from test run speed perspective? What are the comparative execution time numbers on CI for this version vs the main?

My understanding is we'd set `process.env.UPDATE_CHECK_URL` and `process.env.FORCE_UPDATE` only in [the test](https://github.com/TryGhost/Ghost/pull/16840/files#diff-33cb15986ef23b4b684cbd19cffab0b441cee210f8d07dbb8073072e8ac264fcR29-R31) itself - why not use these two environmnental variables as a condition to initialize the update check? 😜

Reply:
No need for these env vars now we have switched to using the job manager directly - Instead we pass the the info in. Have made it so if `forceUpdate` is explicitly defined, the environment check is ignored.